### ISSUE TITLE
Higher granular dials in Sfxr can open a world of sounds

### ISSUE THREAD

**Issue by musikBear** [LMMS/lmms#5404]
### Enhancement Summary
Adding a multiplier for the pitch-slide array in Sfxr will make this generator able to create significantly more versatile output
#### Justification
more sounds from this plugin

#### Mockup

I have fooled around with Sfxr, mostly because i really never paid it much attention. That is an oversight. SFxr has amazing possibilities.
In this picture i have settings for a sweetspot for all 4 dials -I include the preset
![image](https://user-images.githubusercontent.com/6662066/75182015-ccdfda00-573f-11ea-8d06-98483478aa75.png)
Changing any of the framed dial-values *3. decimal*  will result in  extreme changes of output.
If we had a fourth or perhaps even a fifth decimal-option,  a tremendous sound-scape would be available
For demo change START .322 to .323 !
Afair, there is no LMMS plugin that shows that kind of dramatic change in output, when a dial value is changed in 3. decimal.
[sfxrSweetspots.zip](https://github.com/LMMS/lmms/files/4246328/sfxrSweetspots.zip)

**Comment by DomClark** [comment]
Related: #4565

**Comment by zonkmachine** [comment]
@musikBear The step size is set by the `0.001` value in these two models. Poke there.

https://github.com/LMMS/lmms/blob/953a0b17c154b5246d6162ec7c56681e7dc585d4/plugins/sfxr/sfxr.h#L132

https://github.com/LMMS/lmms/blob/953a0b17c154b5246d6162ec7c56681e7dc585d4/plugins/sfxr/sfxr.h#L153

**Comment by musikBear** [comment]
@zonkmachine Great spot! This should be simple -can you assign me to this?

**Comment by musikBear** [comment]
This was a failure
It seams that the point-of-change is not possible to 'catch'
I worked on several dials, but Change| ChangeSpd gave the most understandable result
In stable 1.2.1 we have a super short sound at setting 0.746.
At 0.74**7** the whole sweep through all octaves 
In my Master, i therefore changed the input to 0.0001 that was expected to catch the output from sweetspot 0.746 to 0.746 + 0.000n
And it did. The output was identical with super short sound at 0.746.
That continues to 0.7468, *then* it goes into **whole** sweep through all octaves!
I then added 1 more decimal
At 0.74681 identical with super short sound at 0.746.
At 0.74682 into *whole* sweep through all octaves
Added 2 more decimals..
At 0.7468203 identical with super short sound at 0.746.
At 0.7468204 into whole sweep through all octaves
Looks like This is not possible.
I back out -Sorry
